### PR TITLE
Handled case where the instructor user has no content providers

### PR DESCRIPTION
**Pull request by AndreyLykhoman**
Description: 
 - The handled case where the instructor user has no content providers. Show the page without error.
 - Show message if activities have a disabled content source. 

Tasks: https://vpalresearch.atlassian.net/browse/AD-270

### CODE REVIEW

File: bridge_adaptivity/module/templates/module/collection_detail.html [hunk 1]
```diff
@@ -187,6 +191,13 @@ <h3 class="text-center">Available content courses</h3>
         </button>
       </div>
       <div class="row">
+        {% if not_active_content_source %}
+            <div class="alert alert-danger text-center" role="alert">
+               Something activities have not active content sources.
```
[the remark this answers is not in the document]
Comment (AndreyLykhoman) [reply bridge_adaptivity/module/templates/module/collection_detail.html]: @idegtiarov 
Done.

File: bridge_adaptivity/module/views.py [hunk 2]
```diff
@@ -283,10 +283,13 @@ def get(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         selected_content_sources = list(map(int, self.request.GET.getlist('content_source', [])))
-        activities = Activity.objects.filter(collection=self.object)
+        activities = Activity.objects.filter(collection=self.object).select_related('lti_consumer')
         context = super().get_context_data(**kwargs)
         context['render_fields'] = ['name', 'tags', 'difficulty', 'points', 'source']
         context['activities'] = activities
+        context['not_active_content_source'] = activities.values_list('lti_consumer__name', flat=True).filter(
```
Comment (idegtiarov) [inline bridge_adaptivity/module/views.py]: I would leave only unique lti_consumers' names for the notification message.

**Comment by AndreyLykhoman** [comment]
@idegtiarov 
I fixed nits. Please review one more time.

File: bridge_adaptivity/api/backends/api_client.py [hunk 3]
```diff
@@ -113,7 +109,11 @@ def get_available_courses(request, source_ids=None):
                 )
             )
         except HttpClientError as exc:
-            raise HttpClientError(_("Not valid query: {}").format(exc.message))
+            raise HttpClientError(
+                _("Problem with {} content source. Reason: {} You may ask Bridge administrator to fix this.").format(
```
Comment (idegtiarov) [inline bridge_adaptivity/api/backends/api_client.py]: Please rewrite message a little bit: 
```Please contact Bridge administrator to solve the problem.```

**Comment by AndreyLykhoman** [comment]
@idegtiarov 
I fixed issues. Please review.

**Pull request merged**